### PR TITLE
Add query results window

### PR DESCRIPTION
**Pull request by tsauerwein**
Closes https://github.com/camptocamp/c2cgeoportal/issues/1773

@pgiraud I gave you access rights to my branch, if you want to do some CSS fine tuning.

### CODE REVIEW

File: contribs/gmf/apps/desktop/index.html [hunk 1]
```diff
@@ -110,7 +110,14 @@
             class="dropdown-menu">
           </gmf-backgroundlayerselector>
         </div>
-        <gmf-map class="gmf-map" gmf-map-map="mainCtrl.map"></gmf-map>
+        <gmf-map class="gmf-map" gmf-map-map="mainCtrl.map"
+          ngeo-map-query=""
+          ngeo-map-query-map="::mainCtrl.map"
+          ngeo-map-query-active="mainCtrl.queryActive">></gmf-map>
```
Comment (tsauerwein) [inline contribs/gmf/apps/desktop/index.html]: Thanks!

**Comment by sbrunner** [comment]
Can you purpose an URL to test it?

**Comment by tsauerwein** [comment]
A demo is here: https://tsauerwein.github.io/ngeo/query-results-window/examples/contribs/gmf/apps/desktop/

**Comment by fredj** [comment]
The `Prev` and `Next` are not using the right font-family (it's FontAwesome)

**Comment by tsauerwein** [comment]
> The Prev and Next are not using the right font-family (it's FontAwesome)

Oh yes, the font was different on FF.

**Comment by sbrunner** [comment]
Looks good :-)

**Comment by fredj** [comment]
LGTM

**Comment by tsauerwein** [comment]
Thanks for the reviews!

**Pull request merged**